code/online_challenges/src/rosalind/python_village/conditions_and_loops.py

```diff
@@ -0,0 +1,12 @@
+# Sum all the odd numbers between a and b, inclusive
+def conditionsAndLoops(a, b):
```

Comment:
you could use python built-in loop feature **for i in range(a, b+1)** instead of declaring 'i' as variable and avoiding +1 operation for every even number.